### ISSUE TITLE
Ability to change tree labels using metadata columns

### ISSUE THREAD

**Issue by simonrharris** [jameshadfield/phandango#133]
It would be really nice if it were possible to use metadata columns to label the tree and potentially to add coloured tree nodes based on metadata.

**Comment by jameshadfield** [comment]
Yes! Although there's no information provided via the current formats regarding internal nodes. I know @richardgoater once told me this is possible in phylocanvas.

**Comment by richardgoater** [comment]
There'll be a new Phylocanvas out in a few months, and it should be a bit easier to work with :+1:
